src/operators/timeoutWith.ts

```diff
@@ -43,12 +43,12 @@ class TimeoutWithSubscriber<T, R> extends OuterSubscriber<T, R> {
     return this._hasCompleted;
   }
 
-  constructor(destination: Subscriber<T>,
+  constructor(public destination: Subscriber<T>,
               private absoluteTimeout: boolean,
               private waitFor: number,
               private withObservable: Observable<any>,
               private scheduler: Scheduler) {
-    super(destination);
```

Comment:
I'm unsure, but I don't think we want to do this here, because it may break the unsubscription chain for the "outer" observable. I think what we want to do is add an "InnerSubscriber" that _does_ do this, though.